dtcli/dev.py

```diff
@@ -0,0 +1,31 @@
+import subprocess
+import sys
+import tempfile
+import shutil
+import os
+
+def pack_python_extension(setup_path, target_path, additional_path):
+    with tempfile.TemporaryDirectory() as tmp:
+        args = [sys.executable, '-m', 'pip', 'wheel', '-w', tmp]
+        if additional_path is not None:
+            args.extend(['-f', additional_path])
+
+        args.append(setup_path)
+        result = subprocess.run(args, capture_output=True)
```

Comment:
Can't we avoid using subprocess in favor of using the `pip` module directly? Wouldn't that be simpler, esp. wrt error handling?

Reply:
Pip actively discourages to to be used directly:
https://stackoverflow.com/questions/47059526/should-i-use-pip-main-or-subprocess-call-to-invoke-pip-commands

Reply:
Hmm, it seems it's actually not recommended to use `pip` programmatically, any experience with that?

Reply:
I didn't try it, I've only done a research. It should be possible, but pip can change it's main() interface.
In EXT 1.0 plugins SDK we have also used subprocess to create plugin builds

Reply:
OK, I'm convinced :-)